packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -195,6 +195,7 @@ export interface LightningElement {
     lang: string;
     hidden: boolean;
     draggable: boolean;
+    spellcheck: boolean;
```

Comment:
`spellcheck` is a global attribute, it should be in the base lightning element props.